### PR TITLE
Make the qgisstyle tool fix multiple statements on a single line

### PR DESCRIPTION
**Pull request by SebDieBln**
```
foo(); bar();
```

becomes

```
foo();
bar();
```

[Suggested](https://github.com/qgis/QGIS/pull/2595#discussion_r48018096) by @m-kuhn:

> I think having each command on it's own line makes code much more readable and today's screens fortunately are big enough to show a couple of lines so we don't need to save vertical space ;)

This PR also contains a commit fixing all current instances of this issue.

### CODE REVIEW

File: src/core/qgis.h [hunk 1]
```diff
@@ -53,7 +53,8 @@ class CORE_EXPORT QGis
     //! Used for symbology operations
     // Feature types
     // @deprecated use QgsWKBTypes::Type
-    /* Q_DECL_DEPRECATED */ enum WkbType
+    /* Q_DECL_DEPRECATED */
+    enum WkbType
```
Comment (SebDieBln) [inline src/core/qgis.h]: Could someone please confirm the `Q_DECL_DEPRECATED` still applies to `WkbType`?

File: src/core/gps/gmath.c [hunk 2]
```diff
@@ -252,7 +252,8 @@ int nmea_move_horz(
 
   if ( NMEA_POSIX( isnan )( end_pos->lat ) || NMEA_POSIX( isnan )( end_pos->lon ) )
   {
-    end_pos->lat = 0; end_pos->lon = 0;
+    end_pos->lat = 0;
+    end_pos->lon = 0;
```
Comment (SebDieBln) [inline src/core/gps/gmath.c]: Not sure whether this should be fixed. Is this code coming from some external library?

File: src/server/qgswcsprojectparser.cpp [hunk 3]
```diff
@@ -33,7 +33,8 @@ QgsWCSProjectParser::QgsWCSProjectParser(
 #endif
 )
 #ifdef HAVE_SERVER_PYTHON_PLUGINS
-    : mAccessControl( as )
+    :
+    mAccessControl( as )
```
Comment (SebDieBln) [inline src/server/qgswcsprojectparser.cpp]: Give and take.

**Comment by nyalldawson** [comment]
Nice!

File: src/server/qgswcsprojectparser.cpp (hunk 3 above)
Comment (m-kuhn) [inline src/server/qgswcsprojectparser.cpp]: This #ifdef orgy is not only hard to parse for qgisstyle but also for my brain.

**Comment by m-kuhn** [comment]
Good stuff :+1:
